server/utils/s3.js

```diff
@@ -0,0 +1,45 @@
+const S3 = require('aws-sdk/clients/s3');
+const fs = require('fs');
+
+const bucketName = process.env.AWS_BUCKET_NAME;
+const region = process.env.AWS_BUCKET_REGION;
+const accessKeyId = process.env.AWS_ACCESS_KEY;
+const secretAccessKey = process.env.AWS_SECRET_KEY;
+
+const s3 = new S3({
+    region,
+    accessKeyId,
+    secretAccessKey
+})
+
+function uploadFile(file) {
+    const fileStream = fs.createReadStream(file.path);
+    const uploadParams = {
+        Bucket: bucketName,
+        Body: fileStream,
+        Key: file.filename
+    }
+    return s3.upload(uploadParams).promise();
+}
+exports.uploadFile = uploadFile;
+
+// download a file from s3
+function getFileStream(fileKey) {
```

Comment:
This function used to return a `FileStream` so the name made sense. It does something else now so it might be better to rename it. Once we rename it, we should be able to get rid of the comment as well (i.e. ideally the function name itself should suffice to convey its purpose 👍🏼 ).

Reply:
Totally makes sense, thanks for pointing this 👍.